index.html

```diff
@@ -296,10 +296,11 @@ <h5 class="h-header schedule-speaker__name">Justin Searls</h5>
                                     </div>
                                 </li>
                                 <li class="schedule-session">
-                                    <h3 class="h-header h-trimmed h-schedule-session">Glitch</h3>
+                                    <h3 class="h-header h-trimmed h-schedule-session">Going Front-end to Full-stack with Glitch</h3>
+                                    <img src="img/speakers/Jenn-Schiffer.jpg" alt="Jenn Schiffer" class="schedule-speaker__photo">
```

Comment:
She doesn't have a profile in OpenCFP, hence including the image in the repository.

Reply:
Any way you could size the image down to 250x250?

Reply:
Good call - done!